sdl_viewer/src/lib.rs

```diff
@@ -30,6 +31,9 @@ pub mod gl {
     include!(concat!(env!("OUT_DIR"), "/bindings.rs"));
 }
 
+pub mod box_drawer;
+pub mod color;
```

Comment:
Add a TODO that this is hoisted to the top level crate? We are using the concept of color there too, but it is not yet represented correctly.